src/Communication/docs/Get-AzCommunicationServiceKey.md

```diff
@@ -0,0 +1,145 @@
+---
+external help file:
+Module Name: Az.Communication
+online version: https://docs.microsoft.com/en-us/powershell/module/az.communication/get-azcommunicationservicekey
+schema: 2.0.0
+---
+
+# Get-AzCommunicationServiceKey
```

Comment:
>Key [](start = 28, length = 3)

Seems like this should be "keys", since it returns multiple keys.

Reply:
In azure-powershell we prefer singular name specially for cmdlet name